pybaseball/amateur_draft.py

```diff
@@ -0,0 +1,43 @@
+import pandas as pd
+import requests 
+from bs4 import BeautifulSoup
+import sys
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import QUrl
+from PyQt5.QtWebEngineWidgets import QWebEnginePage
+
+class Client(QWebEnginePage):
+    
+    def __init__(self, url):
+        self.app = QApplication(sys.argv)
+        QWebEnginePage.__init__(self)
+        self.html = ''
+        self.loadFinished.connect(self._on_load_finished)
+        self.load(QUrl(url))
+        self.app.exec_()
+        
+    def _on_load_finished(self):
+        self.html = self.toHtml(self.Callable)
+
+    def Callable(self, html_str):
+        self.html = html_str
+        self.app.quit()
+
+
+def amateur_draft(year, round):
+    round = round.replace('s', '.5')
+    pd.set_option('expand_frame_repr', False)
+    pd.set_option('display.max_columns', None)
+    url = f"https://www.baseballamerica.com/draft-history/mlb-draft-database/#/?Year={year}&Round={round}"
+    page = Client(url)
+    soup = BeautifulSoup(page.html, 'html.parser')
+    draftResults = pd.read_html(soup.prettify())
```

Comment:
why the prettify call?

Reply:
without the prettify call, I get `TypeError: 'NoneType' object is not callable`, so I added it because of https://stackoverflow.com/a/41100562/9790036